doc/whats-new.rst

```diff
@@ -23,6 +23,9 @@ v0.19.1 (unreleased)
 New Features
 ~~~~~~~~~~~~
 
+- Improve repr readability when there are a large number of dimensions in datasets or dataarrays by
+  wrapping the text once the maximum display width has been exceeded. (:issue: `5546`, :pull:`5662`)
+  By `Jimmy Westling <https://github.com/illviljan>`_.
```

Comment:
Isn't that a "breaking change"?

Reply:
I was thinking both are valid but I can move it there.

Reply:
It's fine. I don't have any strong opinions on this really. It makes sense to position it close to your dataset changes,